rust/candid/Cargo.toml

```diff
@@ -38,22 +35,10 @@ thiserror = "1.0.20"
 anyhow = "1.0"
 binread = { version = "2.1", features = ["debug_template"] }
 
-lalrpop-util = { version = "0.20.0", optional = true }
```

Comment:
On line 21, `codespan-reporting = "0.11"` can be removed

Reply:
The `error` submodule still depends on this crate.